src/services/getEditsForFileRename.ts

```diff
@@ -126,8 +126,8 @@ namespace ts {
     ): void {
         const allFiles = program.getSourceFiles();
         for (const sourceFile of allFiles) {
-            const newFromOld = oldToNew(sourceFile.path) as Path;
-            const newImportFromPath = newFromOld !== undefined ? newFromOld : sourceFile.path;
+            const newFromOld = oldToNew(sourceFile.path);

```

Comment:
I think it should be `oldToNew(sourceFile.fileName)`

Reply:
You are right. 
Keen insight!